app/assets/stylesheets/logixRelease.scss

```diff
@@ -1,9 +1,9 @@
-@media only screen and (max-width: 480px) {
-  /* For mobile phones: */
-  [class*="display-4"] {
-    padding-top:10px;
-  }
-}
+// @media only screen and (max-width: 480px) {
+//   /* For mobile phones: */
+//   [class*="display-4"] {
+//     padding-top:10px;
+//   }
+// }
 
```

Comment:
Remove if not needed instead of commenting it out

Reply:
Done